pom.xml

```diff
@@ -146,6 +146,8 @@
     <avro.mapred.classifier>hadoop2</avro.mapred.classifier>
     <jets3t.version>0.7.1</jets3t.version>
     <aws.kinesis.client.version>1.6.2</aws.kinesis.client.version>
+    <!-- Should be consistent with Kinesis client dependency -->
+    <aws.java.sdk.version>1.10.61</aws.java.sdk.version>
```

Comment:
This is probably an ignorant question but is this the first and only time something depends on the AWS SDK here? I know we had discussions about the Kinesis client already because its license was problematic. Didn't it already depend on the AWS SDK and is the license OK? Worth re-checking this situation, if you would, as a second set of eyes would be useful.